runelite-client/src/main/java/net/runelite/client/plugins/feed/FeedPanel.java

```diff
@@ -103,26 +102,42 @@ else if (o2.getType() == FeedItemType.BLOG_POST)
 	}
 
 	private final FeedConfig config;
-	private final Supplier<FeedResult> feedSupplier;
+	private long lastSeenBlogPostTimestamp;
 
-	FeedPanel(FeedConfig config, Supplier<FeedResult> feedSupplier)
+	FeedPanel(FeedConfig config)
 	{
 		super(true);
 		this.config = config;
-		this.feedSupplier = feedSupplier;
+
+		/*
+		 * Use the last seen timestamp from when the panel was created
+		 * as the basis of which blog post items are "new",
+		 * so the items are seen as new even across feed updates
+		 * and across updates to the lastSeenBlogPostTimestamp config item.
+		 */
+		this.lastSeenBlogPostTimestamp = Long.parseLong(config.lastSeenBlogPostTimestamp());
```

Comment:
This should be added to the stringToObject in ConfigManager

Reply:
Good call, updated